### ISSUE TITLE
Testing code

### ISSUE THREAD

**Issue by andi00123456** [hthuwal/sign-language-gesture-recognition#7]
Hi,
     I just got question about testing part. First step: you said moving testing data into TEST folder.
     So, If this test folder is same with previous one which by using train RNN model, then, do I need to create new folder named TEST with my testing video(mp4), at the meantime, I just removed the older one.
    If it's not, then every time we test new video, this code will transfer video into frame every time, including our dataset. I don't think it's necessary. otherwise it will cost a lot.
Please answer me this question.
Thank you

**Comment by andi00123456** [comment]
Because I did test part, it's not work. It just showed me InvalidArgumentError:
InvalidArgumentError (see above for traceback): Unsuccessful TensorSliceReader constructor: Failed to get matching files on /media/../New Volume/data/sign-language-gesture-23/checkpoints_backup/rnn.tflearn: Not found: /media/../New Volume/data/sign-language-gesture-23/checkpoints_backup; No such file or directory
	 [[Node: save_1/RestoreV2 = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_BOOL, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save_1/Const_0_0, save_1/RestoreV2/tensor_names, save_1/RestoreV2/shape_and_slices)]]

Caused by op u'save_1/RestoreV2', defined at:

I didn't figure it out why. I used Second method to predict.
